src/stdlib/SDL_string.c

```diff
@@ -2401,3 +2401,22 @@ int SDL_vasprintf(char **strp, SDL_PRINTF_FORMAT_STRING const char *fmt, va_list
         }
     }
 }
+
+char * SDL_strpbrk(const char *s, const char *accept)
+{
```

Comment:
Is there any convention to call the second argument "accept"? The docs I could find call it "breakset", which also helps understanding the name of the function (which, although small, helps with memorisation).

Reply:
I copied these from [the glibc documentation](https://sourceware.org/git/?p=glibc.git;a=blob;f=string/strpbrk.c;h=d2156c78d2a6a3128d39216b373f2d2fd360c4b5;hb=HEAD#l28).
`breakset` is a good suggestion, and I'll use that.